<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolverSettingsDialog</class>
 <widget class="QDialog" name="SolverSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QToolButton" name="matlab_directory_btn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="output_filename_label">
     <property name="text">
      <string>Output Filename</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QToolButton" name="input_directory_btn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QLabel" name="output_directory">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLabel" name="solver_script">
     <property name="text">
      <string>~/Devel/src/CHANGE/vem_benchmark_2d/matlab/Scripts</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QLineEdit" name="output_filename">
     <property name="text">
      <string>output.txt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="matlab_folder_error">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="solver_script_label">
     <property name="text">
      <string>Solver Script/Executable</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="solver_script_btn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="matlab_folder_label">
     <property name="text">
      <string>Matlab Folder</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="matlab_folder">
     <property name="text">
      <string>~/Matlab_R2019a/bin/matlab</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="output_directory_label">
     <property name="text">
      <string>Output Folder</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QToolButton" name="output_directory_btn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="input_directory_label">
     <property name="text">
      <string>Input Folder</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="input_directory">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="solver_folder_error">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SolverSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SolverSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
